### PR TITLE
add javascript highlighting to README.md

### PR DESCRIPTION
**Pull request by myishay**

### CODE REVIEW

**Comment by myishay** [comment]
@dumbmatter any chance you could review? just a light change to the code highlighting 😃

File: README.md [hunk 1]
```diff
@@ -17,58 +17,65 @@ Originally faces.js was made for [Basketball GM](https://basketball-gm.com/) and
 ## Use
 
 Import it with ES modules:
-
+```javascript
     import { display, generate } from "facesjs";
+```
```
Comment (dumbmatter) [inline README.md:22]: Can you make the spacing consistent, like always have a line break before and after the code blocks?

Comment (myishay) [reply README.md:22]: Sure! I'll make the changes now, please let me know if that is what you meant 🙂

**Comment by dumbmatter** [comment]
Thanks!

**Pull request merged**